client/src/components/PageFormatter.jsx

```diff
@@ -15,6 +16,7 @@ const PageFormatter = () => {
   }
   return (
     <>
+      <Navbar />
```

Comment:
- In a future story, we can probably pass the `user` into the navBar itself. It would avoid the code on conditional rendering on `NavBar` already has.
- No change needed.  Just throwing ideas
```suggestion
      <Navbar user = {user} />
```